src/components/Navbar.tsx

```diff
@@ -24,6 +19,10 @@ const navigation = [
 const profileLinks = [{ name: "Settings", href: "#" }, { name: "Sign out", href: "#" },];
 
 export default function Navbar() {
+  const { data: session } = useSession();
+
+  const user = session?.user;
```

Comment:
To avoid using `?` in all other places, I think we can say this sessions must exits, because if it doesn't, the user will be redirected to the login page.

To enforce this value you can

```suggestion
  // session MUST exists
  const { user } = session!;
```

Reply:
Shoul we use ternary operators and optional chaining with type insertion? This feels awkward but I don't really know how to deal with this.

![image](https://github.com/podcodar/app/assets/107960652/4b95c24f-ecf9-47b4-a864-fcc061378248)